Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -548,12 +554,6 @@ public final class AppDelegateViewModel: AppDelegateViewModelType, AppDelegateVi
     self.crashManagerDidFinishSendingCrashReportProperty.signal
       .observeValues { AppEnvironment.current.koala.trackCrashedApp() }
 
-    self.applicationLaunchOptionsProperty.signal
```

Comment:
This event is in the merged signal that triggers the config fetch so we didn't need this anymore. One thing to note is that this did have `take(first: 1)` so it was explicitly only meant to happen once per app session. I can't really think of any reason that it would be problematic to do this many times per session (backrounding/foregrounding) but worth considering!